skyve-war/src/main/java/modules/admin/Group/GroupService.java

```diff
@@ -0,0 +1,60 @@
+package modules.admin.Group;
+
+import java.util.List;
+
+import org.skyve.domain.Bean;
+import org.skyve.persistence.DocumentQuery;
+import org.skyve.persistence.Persistence;
+
+import jakarta.enterprise.inject.Default;
+import jakarta.inject.Inject;
+import modules.admin.domain.Group;
+
+/**
+ * This class acts as a service layer to encapsulate domain logic.
+ *
+ * Add this line to classes that wish to use it: @Inject private transient GroupService groupService;
+ */
+@Default
+public class GroupService {
+	@Inject
+	private Persistence persistence;
```

Comment:
I think the Skyve injected classes are already safe from serialisation, but we should declare these as `private transient` just in case, and it reinforces the pattern for other developers to copy when injecting classes.